src/modules/login/views/ViewLogin.vue

```diff
@@ -0,0 +1,160 @@
+<script>
+import validateEmail from "@/utils/validateEmail";
+import validatePassword from "@/utils/validatePassword.js";
+import BaseInput from "@/components/BaseInput.vue";
+import BaseAlert from "@/components/BaseAlert.vue";
+import { loginUser } from "../services";
+import axios from "axios";
```

Comment:
why are you importing Axios?

Reply:
update: removed the import, was used for testing